modules/nodes-pods-autoscaling-custom-metrics-access.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-autoscaling-custom.adoc
+// Modeled after migration-accessing-performance-metrics.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-metrics-access_{context}"]
+= Accessing performance metrics
+
+The Custom Metrics Autoscaler Operator records metrics and pulls them into on-cluster monitoring storage. You can query the metrics by using Prometheus Query Language (PromQL) to diagnose migration performance issues. All metrics are reset when the Migration Controller pod restarts.
+
+You can access the performance metrics and run queries by using the {product-title} web console.
+
+.Procedure
+
+. In the {product-title} web console, click *Observe* -> *Metrics*.
+. Enter a PromQL query, select a time window to display, and click *Run Queries*.
```

Comment:
Does it make sense to provide a ready-to-use query as an example?

Reply:
It might. Do you have one we could use?